cluster/charts/rook-ceph/values.yaml.tmpl

```diff
@@ -29,6 +29,9 @@ nodeSelector:
 # Tolerations for the rook-ceph-operator to allow it to run on nodes with particular taints
 tolerations: []
 
+# Whether rook watches its current namespace for CRDs or the entire cluster, defaults to true
+rookCurrentNamespaceOnly: true
```

Comment:
I'd suggest to default to `false`. Defaulting to `true` has really just caused heartburn for some users.